docs/sql_reference/src/asciidoc/_chapters/sql_statements.adoc

```diff
@@ -5958,6 +6098,122 @@ GRANT SELECT (part_no, part_name), DELETE ON TABLE invent.partloc
 GRANT SELECT ON TABLE invent.partloc TO ajones;
 ```
 
+* This example explains how to grant the `SELECT` privilege to PUBLIC.
+
++
+The _testuser1_ creates the table _t1_. The _testuser2_ and _testuser3_ do not have the `SELECT` privilege on the table _t1_.
+
++
+_testuser2_:
+
++
+```
+SQL>SELECT * FROM t1;
+
+*** ERROR[4481] The testuser2 does not have SELECT privilege on table or view TRAFODION.SEABASE.T1. [2018-06-11 11:39:16]
+```
+
++
+The _testuser1_ grants the `SELECT` privilege on table _t1_ to PUBLIC, which means granting `SELECT` privilege to all users (_testuser2_ and _testuser3_). 
+

```

Comment:
Word smith suggestion:  The _testuser1_ grants the ... privilege to all current  and future users (_testuser2_ and _testuser3_).

Reply:
Thanks @robertamarton, your comment has been incorporated :pencil2: